<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database</class>
 <widget class="QDialog" name="Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>587</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>798</width>
    <height>587</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>361</width>
       <height>31</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="ButtonApply">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonCancel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonOK">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>410</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="m_pSections">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>Textures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="m_pTextureTabs">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="DatabaseWidgetTile" name="m_pTiles">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <attribute name="title">
           <string>Tiles</string>
          </attribute>
         </widget>
         <widget class="DatabaseWidgetTileSet" name="m_pTileSets">
          <attribute name="title">
           <string>TileSets</string>
          </attribute>
         </widget>
         <widget class="DatabaseWidgetAutoTile" name="m_pAutoTiles">
          <attribute name="title">
           <string>AutoTiles</string>
          </attribute>
         </widget>
         <widget class="DatabaseWidgetSprite" name="m_pSprites">
          <attribute name="title">
           <string>Sprites</string>
          </attribute>
         </widget>
         <widget class="DatabaseWidgetAnimation" name="m_pAnimations">
          <attribute name="title">
           <string>Animations</string>
          </attribute>
         </widget>
         <widget class="DatabaseWidgetAnimationType" name="m_pAnimationTypes">
          <attribute name="title">
           <string>AnimationTypes</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Objects">
      <attribute name="title">
       <string>Objects</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTabWidget" name="m_pObjectTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="m_pWorldObjects">
          <attribute name="title">
           <string>WorldObjects</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="m_pDynamicObjects">
          <attribute name="title">
           <string>DynamicObjects</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Text">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="m_pTextTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="DatabaseWidgetLocalisation" name="m_pTexts">
          <attribute name="title">
           <string>Localisations</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatabaseWidgetLocalisation</class>
   <extends>QWidget</extends>
   <header>databasewidgetlocalisation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatabaseWidgetTileSet</class>
   <extends>QWidget</extends>
   <header>databasewidgettileset.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatabaseWidgetSprite</class>
   <extends>QWidget</extends>
   <header>databasewidgetsprite.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatabaseWidgetTile</class>
   <extends>QWidget</extends>
   <header>databasewidgettile.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatabaseWidgetAnimationType</class>
   <extends>QWidget</extends>
   <header>databasewidgetanimationtype.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatabaseWidgetAnimation</class>
   <extends>QWidget</extends>
   <header>databasewidgetanimation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatabaseWidgetAutoTile</class>
   <extends>QWidget</extends>
   <header>databasewidgetautotile.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_pSections</tabstop>
  <tabstop>m_pTextureTabs</tabstop>
  <tabstop>ButtonOK</tabstop>
  <tabstop>ButtonCancel</tabstop>
  <tabstop>ButtonApply</tabstop>
  <tabstop>m_pTextTabs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
